src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/validation/YFlowValidator.java

```diff
@@ -153,6 +154,20 @@ private void checkBandwidth(YFlowRequest yFlowRequest)
         }
     }
 
+    private void checkMaxLatency(YFlowRequest yFlowRequest) throws InvalidFlowException {
+        if (yFlowRequest.getMaxLatency() == null || yFlowRequest.getMaxLatencyTier2() == null) {
+            return;
+        }
+
+        if (yFlowRequest.getMaxLatency() > yFlowRequest.getMaxLatencyTier2()) {
```

Comment:
Let's throw exeption if maxlateny >= tier2. because it has no sense if they are equal

Reply:
done